docs/src/ReactPlayground.js

```diff
@@ -1,5 +1,5 @@
 var React = require('react');
-var classSet = require('react/lib/cx');
+var classSet = require('classNames');
```

Comment:
You have a Mac and thus a case-insensitive filesystem :grin:. This should be 'classnames' right?